Client/Frontend/UserContent/UserScripts/CookieControl.js

```diff
@@ -0,0 +1,40 @@
+// The $<> is used to replace the var name at runtime with a random string.
```

Comment:
Should add the license header here, even though it is JS, it is original code from us.